### DISCUSSION TITLE
Too many task icons on the taskbar

### DISCUSSION THREAD

**Discussion by stardepp** [hluk/CopyQ#2670] [Q&A] [answered]
I have too many task icons on the taskbar in CopyQ. How can I remove them?

![2024-04-10 04_17_43-_BILD_ - CopyQ](https://github.com/hluk/CopyQ/assets/40627059/1d2b1f44-4d30-44a9-bfe8-e2daccfa9cd0)

**Comment by hluk** [comment, marked as answer]
To remove some of the icons: Open Commands dialog (F6 shortcut) and uncheck unneeded items.

![image](https://github.com/hluk/CopyQ/assets/38054/8d1a1a75-14ab-40ca-8b6d-eedd830e9846)

**Reply by stardepp** [reply]
Thank you very much for your help, I'm delighted.
